### DISCUSSION TITLE
Manter valores recebidos via PJ em dólar ou enviar valores para conta PF Europa

### DISCUSSION THREAD

**Discussion by diemesleno** [DyegoCosta/trabalhando-remoto#189] [General]
Pessoal,

Sou sócio de uma empresa exportadora de serviços e recebemos os valores em dólar através do Payoneer. 

Até o momento temos transferido do Payoneer para a conta PJ da empresa, e neste caso o Payoneer realiza a conversão, uma taxa é paga e uma empresa bancária parceira do Payoneer faz o depósito dos valores já em reais na conta bancária da empresa.

Dos valores recebidos, após pago os impostos da empresa, o lucro é repartido entre os sócios e os valores são transferidos para as contas pessoais de cada sócio.

No meu caso eu recebo os valores em reais na minha conta e parte deste valor (a maior parte) eu envio para uma conta bancária que tenho na Europa. Neste caso tenho que utilizar TransferWise ou Remessa Online para converter para Euro e enviar.

O problema é que desta forma estou perdendo 2 vezes em taxas e conversões (Dólar -> Reais e Reais -> Euro)

Minha dúvida é se é possível, de forma legal, enviar os valores (minha parte) já do Payoneer para a conta bancária pessoal na Europa ou se é a empresa é obrigada a receber os valores na conta da empresa no Brasil para somente depois do pagamento dos impostos e divisão dos lucros seja possível enviar os valores.

Não sei se conseguiram me compreender, mas desde já muito obrigado.

**Comment by clmadrazo** [comment]
Desde a conta da empresa no Payoneer podem transferir para uma conta Payoneer de pessoa física e aí já você transferir para sua conta na Europa.

**Comment by diemesleno** [comment]
Olá @clmadrazo,

Muito obrigado pela interação!

Mas ai como ficaria a declaração de IR? Eu continuaria fazendo entrada dos valores normalmente através da PTAX do dia mesmo que o dinheiro não tenha entrado no Brasil? 

Ou seja, no momento que a empresa receba os valores em dólares no Payoneer eu já posso considerar isso como a entrada de dinheiro na empresa informando que foi recebido por exemplo Valor x PTAX e desta forma calcular os impostos a serem pagos e da mesma forma fazer a divisão dos lucros já em dólares e enviar os valores para a conta de cada sócio na Europa?

**Comment by clmadrazo** [comment]
Aí já é com vocês né kkkk…

Não tenho conhecimento até o presente momento de que o Payoneer declare alguma coisa para a Receita se os valores não são transferidos pro Brasil (tenho pesquisado bastante sobre isso e não tenho encontrado nada que indique o contrário). O comum é que a pessoa declare os recursos que recebe no Brasil e no final do ano os valores em contas no exterior. 

Espero ter ajudado ;)

**Comment by diemesleno** [comment]
Olá @clmadrazo,

Então, ai que está o problema. Porque não quero deixar nada ilegal. 

Só quero não precisar fazer o processo de USD -> BRL -> EUR
pois neste caso perco muito nas duas pontas.

Queria deixar o processo como USD -> EUR

Em todo caso muito obrigado!

**Comment by clmadrazo** [comment]
Eu não falei que fosse ilegal, realmente nem é kkkk.

Você pode continuar declarando como mesmo vinha fazendo mesmo que o valor nunca entre no Brasil. Ao final é um valor que você ganhou, então entra da mesma forma.

Abraço

**Comment by copercini** [comment]
Também acho que você pode fazer igual ao @clmadrazo falou, e não esquecer de declarar a conta europeia no imposto de renda. Mas isso é só um palpite, a melhor coisa seria falar com o seu contador

**Comment by iwalucas** [comment]
@diemesleno tenho a mesma dúvida. Hoje vc emite a NF com base no ptax venda do dia, ou valor liquido em real que cai na conta?

Eu entendo que ao utilizar o valor ptax, tanto faz o dinheiro de fato ter entrado na conta PJ no Brasil, uma vez que a conta do payoneer eh PJ, ou seja, um bem da empresa que reconhece a receita.

Nesse caso o valor poderia ir pra Europa diretamente, mas eh apenas um achismo, o ideal seria ter alguma posição oficial. Vou falar com meu contador e ver se é possível.

**Comment by diemesleno** [comment]
Olá @iwalucas 
Eu não emito NF porque a empresa na qual eu presto serviço através da minha empresa é estrangeira,
e eles não necessitam da NF. Mas eu informo para minha contadora os valores recebidos em reais na conta da empresa. Ou seja, o líquido. 
Mas a partir já do próximo mês irei informar o valor baseado na PTAX do dia que receber e irei enviar já os valores para as contas pessoas na Europa.
Em todo caso estou em contato com minha contadora também que está se iterando das possibilidades.
Abraço!

**Comment by leomsaippa** [comment]
Olá @diemesleno você conseguiu resolver este problema? Estou com uma oportunidade de receber em dólar e queria saber se eu tenho que pagar imposto no que vou transferir direto para minha corretora americana.

**Comment by diemesleno** [comment]
Olá @leomsaippa consegui resolver sim.

No caso estou fazendo isso, ao receber os valores em dólar na Payoneer e de lá transfiro para minha conta em EUR na Alemanha. Para a minha contadora eu informo estes valores recebidos convertidos em reais na cotação PTAX do dia.

Na declaração de IR minha contadora tem que informar a conta e valores que tenho no banco alemão, incluindo o valor convertido para reais na PTAX do último dia 31/12.

**Comment by samiraguiar** [comment]
@diemesleno, pode contar qual banco você usa na Alemanha? Dá pra abrir conta como residente aqui no Brasil ou você abriu essa conta de lá mesmo?

**Comment by diemesleno** [comment]
Olá @samiraguiar estou usando o N26, mas até onde sei não é possível ainda abrir conta nele no Brasil.

Eu continuo no Brasil, mas o que fiz foi aproveitar uma viagem que fiz para Portugal em dezembro passado e 
criar a conta colocando o endereço do hotel que fiquei hospedado. Abri a conta ainda estando no Brasil, uns 5 dias antes da minha viagem e ao chegar no hotel avisei que deveria receber uma carta. Eles receberam e me entregaram. Já era meu cartão com a conta ativa.

Abraço!
